requirements_py3.txt

```diff
@@ -1,6 +1,10 @@
-nose>=1.3.1
-yaswfp>=0.4
-beautifulsoup4==4.6.0
-m3u8==0.3.2
-slimit==0.8.1
-ply==3.4
+beautifulsoup4==4.8.2
+certifi==2019.11.28
+chardet==3.0.4
+defer==1.0.4
+idna==2.9
+pkg-resources==0.0.0
+pyxdg==0.26
+requests==2.23.0
+soupsieve==2.0
+urllib3==1.25.8
```

Comment:
Beyond other comments in this file, this whole file needs to go away. We should have only one requirements file, for Python 3 (ideally, we should have 2, one for "production", the other for the extra dependencies to run the tests, but we can make that separation later).